Assets/MRTK/SDK/Profiles/DefaultXRSDKControllerVisProfile.asset

```diff
@@ -29,7 +29,7 @@ MonoBehaviour:
   globalRightHandVisualizer: {fileID: 3512491064599439886, guid: 7a857af0cb85b4a47b6ec8a1babcb73c,
     type: 3}
   controllerVisualizationSettings:
-  - description: Windows Mixed Reality XRSDK Motion Controller
+  - description: Undefined Controller
```

Comment:
We should make sure this is reserialized in Unity 2019, where these types are loaded. I might also look into not preventing these assemblies from loading on certain Unity versions, since it seems to cause more issues than not!

Reply:
ok, will reserialze there